### PR TITLE
Story/io 18/project phase to planning list

### PR DESCRIPTION
**Pull request by EngbergJonas**
- Phase Dialog component to groups list
- Patching of project phase from list view
- Updated projects list in redux when updating a project
- useClickOutsideRef-hook

### CODE REVIEW

File: src/components/Planning/PhaseDialog/PhaseDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <div className="phase-dialog-footer">
+        <Button size="small" onClick={handleSubmit(onSubmit)}>
+          {t('save')}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+export default PhaseDialog;
```
[the remark this answers is not in the document]
Comment (kgunga) [reply src/components/Planning/PhaseDialog/PhaseDialog.tsx:80]: Dialog should be opened within View port

**Review by kgunga — CHANGES_REQUESTED** [review]

File: src/components/Planning/PlanningGroupsTable/PlanningGroupsTableRow.tsx [hunk 2]
```diff
@@ -47,7 +59,14 @@ const PlanningGroupsTableRow: FC<IPlanningProjectsTableProps> = ({ project }) =>
         <div className="project-header-cell-container">
           {/* LEFT */}
           <div className="left">
-            <IconMenuDots size="xs" />
+            <IconMenuDots
+              size="xs"
+              style={{ cursor: 'pointer' }}
```
Comment (kgunga) [inline src/components/Planning/PlanningGroupsTable/PlanningGroupsTableRow.tsx:64]: Please use css styling instead of direct styling

**Review by kgunga — CHANGES_REQUESTED** [review]

File: src/components/Planning/PhaseDialog/PhaseDialog.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (EngbergJonas) [reply src/components/Planning/PhaseDialog/PhaseDialog.tsx]: resolved

File: src/components/Planning/PlanningGroupsTable/PlanningGroupsTableRow.tsx (hunk 2 above)
Comment (EngbergJonas) [reply src/components/Planning/PlanningGroupsTable/PlanningGroupsTableRow.tsx]: resolved

**Review by kgunga — APPROVED** [review]
👍

**Pull request merged**